tools/provision-alerts/template/template.yml

```diff
@@ -350,4 +377,59 @@ policies:
         operator: "above"
         template_name: "Generic metric comparator"
 
+      - name: Process / Process count
+        threshold: 0
+        duration: 10
+        operator: "above"
+        template_name: "Process count comparator"
+
+      - name: Process / Username empty
+        threshold: 0
+        duration: 5
+        operator: "above"
+        template_name: "Process username empty"
+
+      - name: Agent process / Cpu Percent
+        metric: cpuPercent
+        threshold: 0.1 # max 0.05 in last week
+        duration: 10
+        operator: "above"
+        template_name: "Agent process metrics"
+
+      - name: Agent process / Cpu System Percent
+        metric: cpuSystemPercent
+        threshold: 0.05 # max 0.02 in last week
+        duration: 10
+        operator: "above"
+        template_name: "Agent process metrics"
+
+      - name: Agent process / Cpu User Percent
+        metric: cpuUserPercent
+        threshold: 0.06 # max 0.02 in last week
+        duration: 10
+        operator: "above"
+        template_name: "Agent process metrics"
+
+      - name: Agent process / IO Read Bytes Per Second
```

Comment:
would those alerts catch the scenario when there is no data for those metrics?

Reply:
if the new release reports no data it will trigger as the previous should be reporting. Still it would be simple to add another template and add more alerts. WDYT?